.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+*Short paragraph summarizing the changes*
+
+---
+
+Thank you for submitting a PR to xDSL!
+
+Make sure to follow the steps in the checklist below *before* you submit this PR!
+
+## Checklist before you submit this PR:
+
+ - [ ] If your PR is not ready for review, open it as a draft instead.
+ - [ ] Make sure your PR title follows the [guidelines](https://github.com/xdslproject/xdsl/wiki/Tags-for-commit-messages-and-PR-titles)
+ - [ ] Add some tags to categorize the PR
```

Comment:
This does not work for new users, should we change the text to reflect that?

Reply:
I think as soon as their title is fine, the person who will click the button to merge should be the one to hold responsibility for the label/tag being there